app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,76 @@
+class OrdersController < ApplicationController
+  rescue_from ActiveRecord::RecordNotFound, with: :catch_not_found
+  before_action :set_order, only: %i[ show edit update destroy ]
+
+  # GET /orders or /orders.json
+  def index
+    @orders = Order.all
+  end
+
+  # GET /orders/1 or /orders/1.json
+  def show
+  end
+
+  # GET /orders/new
+  def new
+    @order = Order.new
+    @customers = Customer.all
+  end
+
+  # GET /orders/1/edit
+  def edit
+    @customers = Customer.all
```

Comment:
If you were using `@customers` to populate the customers dropdown, then this would be fine, but `@customers` isn't referenced anywhere in `app/views/orders`! These actions are loading the entire table for nothing.